libraries/AP_Proximity/AP_Proximity_RPLidarA2.cpp

```diff
@@ -63,6 +63,17 @@
 
 extern const AP_HAL::HAL& hal;
 
+AP_Proximity_RPLidarA2::AP_Proximity_RPLidarA2(AP_Proximity &_frontend, AP_Proximity::Proximity_State &_state) :
+    AP_Proximity_Backend_Serial(_frontend, _state)
+{
+    // we can fill in all 72 distances:
+    obstacle_distance.increment = 5;
```

Comment:
let's fille in increment_f here as well I think..

Reply:
Done.